src/hooks/index.ts

```diff
@@ -115,3 +115,8 @@ export const useUserFollowing = () : UserFollowing => {
 
     return { following, onFollow, onUnfollow };
 };
+
+export const useFocusDate = (): [Date, Dispatch<SetStateAction<Date>>] => {
+    const [focusDate, setFocusDate] = useState(new Date(Date.now()));
+    return  [focusDate, setFocusDate];
```

Comment:
I'm wondering if it wouldn't be better to return these in an object instead of an array. That way we could potentially add something down the line if we feel the need to, without having to change the API.